src/components/Todo.jsx

```diff
@@ -47,9 +49,9 @@ export default function Todo({
         type="button"
         onClick={() => dispatch({ type: REMOVE_TODO, payload: { id } })}
       >
-        X
+       <RiDeleteBinFill />
       </button>
-      <button onClick={() => setIsEditing(true)}>Edit</button>
+      <button onClick={() => setIsEditing(true)}><RiEdit2Fill /></button>
```

Comment:
Could we change the order of this icons. So right now you can see, from left to rigth, first the `delete icon` and then the `edit-icon`. I think that is a bit weird. I should be first edit and then delete. What do you think?

Reply:
done...you are right